<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QPushButton" name="runButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Inicio</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="serialPortLabel">
    <property name="text">
     <string>Puerto Serie:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="serialPortComboBox"/>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>64</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Estado:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>14</y>
     <width>331</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Detenido</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>951</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPushButton" name="ADCButton">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>80</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>LeeADC</string>
     </property>
    </widget>
    <widget class="QwtKnob" name="Knob">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>241</width>
       <height>191</height>
      </rect>
     </property>
     <property name="upperBound">
      <double>1.000000000000000</double>
     </property>
     <property name="value">
      <double>0.500000000000000</double>
     </property>
     <property name="singleSteps">
      <UInt>0</UInt>
     </property>
     <property name="pageSteps">
      <UInt>10</UInt>
     </property>
    </widget>
    <widget class="QSplitter" name="splitter_2">
     <property name="geometry">
      <rect>
       <x>32</x>
       <y>56</y>
       <width>97</width>
       <height>66</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QCheckBox" name="rojo">
      <property name="text">
       <string>Rojo</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="verde">
      <property name="text">
       <string>Verde</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="azul">
      <property name="text">
       <string>Azul</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pingButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Ping</string>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdCh1">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdCh2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>120</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdCh3">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>40</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdCh4">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>120</y>
       <width>261</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>PWM/GPIO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BotonSondeo">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>380</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sondeo</string>
     </property>
    </widget>
    <widget class="color_widgets::ColorWheel" name="colorWheel">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>230</y>
       <width>231</width>
       <height>161</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="Led" name="BotonIzq">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>240</y>
       <width>151</width>
       <height>131</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="Led" name="BotonDer">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>240</y>
       <width>191</width>
       <height>131</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Color Led component</string>
     </property>
     <property name="whatsThis">
      <string>Led indicator</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="EnableSondeo">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>370</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>ACTIVAR SONDEO</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <widget class="QwtPlot" name="Grafica">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>941</width>
       <height>411</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="canvasBackground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="autoReplot">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Tab 3</string>
    </attribute>
    <widget class="QwtPlot" name="GraficaDigital">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>941</width>
       <height>411</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="canvasBackground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="autoReplot">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="Actualizar">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>410</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
    <widget class="QwtCounter" name="Counter">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>410</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="x10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>520</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>x10</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="MODO">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>560</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>MODO RTC  /  OSCILOSCOPIO</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="x100">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>550</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>x100</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="STOP">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>520</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="x1000">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>580</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>x1000</string>
   </property>
  </widget>
  <widget class="QwtKnob" name="frecuencia">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="upperBound">
    <double>5.000000000000000</double>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QWidget</extends>
   <header>analogwidgets/led.h</header>
  </customwidget>
  <customwidget>
   <class>color_widgets::ColorWheel</class>
   <extends>QWidget</extends>
   <header>color_wheel.hpp</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
  <customwidget>
   <class>QwtKnob</class>
   <extends>QWidget</extends>
   <header>qwt_knob.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>serialPortComboBox</tabstop>
  <tabstop>pingButton</tabstop>
  <tabstop>runButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
